frontend/views/components/group-creation-steps/GroupPurpose.vue

```diff
@@ -3,17 +3,20 @@
   i18n.is-title-4.steps-title(tag='h4') 2. Group Purpose
 
   .card
-    i18n.label(tag='label') How would you describe your group?
+    label.field
+      i18n.label How would you describe your group?
+
+      textarea.textarea(
+        name='sharedValues'
+        ref='purpose'
+        :placeholder='L("Group Purpose")'
+        maxlength='500'
+        :class='{ error: $v.form.sharedValues.$error }'
+        :value='group.sharedValues'
+        @input='update'
+      )
+      i18n.helper This is optional.
```

Comment:
Should we add (or modify) a cypress test to create a group without description?
We already have a test to update the description.

Reply:
A test to see if the group is created without a description? Sure, if you think that's usefull

Reply:
Yes, I believe it's a very important test, mainly because the contract was changed in this PR. If the change you made at `group.js` is reverted in the future, this e2e test will catch it. And also because our current "error message" in the group creation isn't very explicit.

Reply:
Ok, will do a test to see if a group can be created without a description